cptv-reprocess.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
+
+import argparse
+
+from api import API
+
+
+class CPTVReprocessor:
+    def __init__(self, server_url, user, password, id, limit):
+        self.user = user
+        self.password = password
+        self.url = server_url
+        self.recording_id = id
+        self.limit = limit
+
+    def reprocess(self):
+        """ Reprocesses all specified recording from specified server """
+
+        api = API(self.url, self.user, self.password)
+
+        print(f"Querying server {self.url}")
+        print(f"Limit is {self.limit}")
+        if self.recording_id:
+            print(f"Recording id is {self.recording_id}")
+            api.reprocess([self.recording_id])
+            return
+
+        where = {}
+        # algorithm was only added when trackbased tagging was introduced
+        where["additionalMetadata.algorithm"] = {"$eq": None}
+        where["type"] = "video"
```

Comment:
how about just:

```python
where = {
   "additionalMetadata.algorithm" = {"$eq": None},
   "type": "video",
}
```

Reply:
Also, the type isn't `video` it's, `thermalRaw`

Reply:
seems to work with either or, i just used video as that's what browse sends